packages/salesforcedx-vscode-soql/src/editor/soqlEditorInstance.ts

```diff
@@ -101,15 +143,60 @@ export class SOQLEditorInstance {
           this.updateTextDocument(document, e.message);
           break;
         }
+        case MessageType.SOBJECT_METADATA_REQUEST: {
+          this.retrieveSObject(e.message).catch(() => {
+            channelService.appendLine(
+              `An error occurred while handling a request for object metadata for the ${
+                e.message
+              } object.`
+            );
+          });
+          break;
+        }
+        case MessageType.SOBJECTS_REQUEST: {
+          this.retrieveSObjects().catch(() => {
+            channelService.appendLine(
+              `An error occurred while handling a request for object names.`
+            );
+          });
+          break;
+        }
```

Comment:
These are asynchronous calls. A nested web service request may be necessary so we don't `await`. Otherwise may block unnecessarily.

Reply:
Yeah I'm right with you on the fence on this one. These async calls actually have try-catch blocks in the called function, and these catch blocks are mostly called to satisfy the linter.